### PR TITLE
test: add container related util functions

### PR DESCRIPTION
**Pull request by Letty5411**
Signed-off-by: letty <letty.ll@alibaba-inc.com>

**1.Describe what this PR did**
Add container related functions for test.

**2.Does this pull request fix one issue?** 

It now prints more info when the response status is not as expected.
**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

**Comment by allencloud** [comment]
How about directly using `CreateBusyboxContainer` instead of `CreateBusyboxCon`? 🌹

**Comment by Letty5411** [comment]
OK

File: test/utils.go [hunk 1]
```diff
@@ -20,3 +26,90 @@ func SkipIfFalse(c *check.C, conditions ...VerifyCondition) {
...
 	}
 }
+
+// CreateBusyboxContainer creates a basic container using busybox image.
+func CreateBusyboxContainer(c *check.C, cname string, cmd ...string) {
+	q := url.Values{}
+	q.Add("name", cname)
+
+	// If not specified, CMD executed in container is "top".
+	if cmd == nil {
```
Comment (allencloud) [inline test/utils.go]: len(cmd) == 0

File: test/utils.go [hunk 2]
```diff
@@ -20,3 +26,131 @@ func SkipIfFalse(c *check.C, conditions ...VerifyCondition) {
...
+	c.Assert(err, check.IsNil)
+	c.Assert(resp.StatusCode, check.Equals, 200)
+
+	defer resp.Body.Close()
+
+	got := types.ContainerJSON{}
+	err = request.DecodeBody(&got, resp.Body)
+	c.Assert(err, check.IsNil)
+
+	return string(got.State.Status) == status
```
Comment (allencloud) [inline test/utils.go]: It may panic, since `State *ContainerState`. We had better detect State is non-nil first.

File: test/utils.go (hunk 1 above)
Comment (Letty5411) [reply test/utils.go]: ok~

**Comment by allencloud** [comment]
LGTM

**Pull request merged**